docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -111,8 +115,12 @@ To fully onboard your repository you may take the following steps:
     * Simplifying the configuration to fewer declarations by using broader package ID prefixes or [setting a default source](#setting-default-sources) where possible.
     * Verifying the source each package was restored from by checking the [metadata files in the global packages folder or reviewing the restore logs](https://devblogs.microsoft.com/nuget/performance-and-polish-with-nuget-5-9/).
 
-For an idea of how your source mappings may look like, refer to our [samples repo](https://github.com/NuGet/Samples/tree/main/PackageSourceMappingExample).
+#### Automated onboarding using tool
 
+For large solution with a many packages above manual onboarding could be time consuming and error prone. We developed [NuGet.PackageSourceMapper tool](https://www.nuget.org/packages/NuGet.PackageSourceMapper) automatically generate a NuGet.config for you based on your project’s known packages and sources. The package source mapper tool requires you to have completed a successful package restore in which it will read each respective `.nupkg.metadata` file generated as part of your build to best understand how you map your respective packages and sources. Tool not only covers top dependencies it also considers all the transitive dependencies when generating mapping.

```

Comment:
Consider entering a new line after the second sentence (basically before you have the text from the blog).  That'd improve readability as it doesn't look like a complete block.

The first 2 sentences have some small errors. 

How about

> Many repositories have a large number of packages and doing the work manually can be time consuming. The [NuGet.PackageSourceMapper tool](https://www.nuget.org/packages/NuGet.PackageSourceMapper) can automatically generate a NuGet.config for you, based on your project's known packages and sources.

Reply:
Sure, addressed 2 PR comments.